app/services/invoices/payments/stripe_service.rb

```diff
@@ -202,21 +202,15 @@ def handle_missing_payment(organization_id, stripe_payment)
         result
       end
 
-      # NOTE: Due to RBI limitation, all indians payment should be off_session
-      # to permit 3D secure authentication
-      # https://docs.stripe.com/india-recurring-payments
-      def off_session?
+      # NOTE: Due to RBI limitation, all indians payment should be "on session". See: https://docs.stripe.com/india-recurring-payments
+      # crypto payments don't support 'off_session'
+      def setup_future_usage?
```

Comment:
Don't know if it would be useful for others but I guess a comment with a link to https://docs.stripe.com/api/payment_intents/create#create_payment_intent-setup_future_usage might be useful.